Mage/src/main/java/mage/collation/Rotater.java

```diff
@@ -15,7 +15,7 @@
 public class Rotater<T> {
 
     private final List<T> items;
-    private int position;
+    private int position, stripeLen = 0, stripeWidth, stripeDepth;
```

Comment:
The code to implement striped collation almost certainly needs to be a separate class, perhaps both implementing a common interface.

You're going to have a hard time getting this to work without a dev environment for testing.